tests/unittests/JITTest.cpp

```diff
@@ -561,6 +561,78 @@ TEST_P(BackendCorrectnessTest, smallConv) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+TEST_P(CPUOnly, convDKKC8Test) {
+  ExecutionEngine EE_(backendKind_);
+  auto &mod_ = EE_.getModule();
+  auto *F_ = mod_.createFunction("main");
+
+  auto *input = mod_.createVariable(ElemKind::FloatTy, {1, 2, 1, 32}, "input");
+  auto IH = input->getHandle();
+  for (size_t i = 0; i < 2 * 32; i++) {
+    IH.raw(i) = i + 1;
+  }
+
+  auto filter =
+      mod_.createVariable(ElemKind::FloatTy, {128, 1, 1, 16}, "filter");
+  auto FH = filter->getHandle();
+  for (size_t i = 0; i < 128; i++)
+    for (size_t j = 0; j < 16; j++) {
+      FH.at({i, 0, 0, j}) = (i + j) / 100.0;
+    }
+
+  auto *zeroBias = mod_.createVariable(ElemKind::FloatTy, {128}, "bias");
+  zeroBias->getPayload().zero();
+
+  auto outTy = mod_.uniqueType(ElemKind::FloatTy, {1, 2, 1, 128});
+
+  ConvolutionNode *CN =
+      F_->createConv("Conv", input, filter, zeroBias, outTy, 1, 1, 0, 2);
+  SaveNode *S = F_->createSave("save", CN);
+
+  EE_.compile(CompilationMode::Infer, F_);
+  EE_.run({}, {});
+
+  auto result = llvm::cast<Variable>(S->getOutput())->getPayload().getHandle();
+
+  std::array<float, 256> ref = {
```

Comment:
Why is this necessary? The results should be the same for the Interpreter's group conv, so you should be able to follow the pattern of the other tests here in JITTest -- run the same group conv on the Interpreter and the `backendKind_ ` and then `EXPECT_TRUE(out1.isEqual(out2));`.

Reply:
I tried to follow the tests pattern in JITTest, passing the Tensors and the backendKind_. However, when I tried to get VarFrom(Tensor), the visibilityKind is public, which is necessary for the function ExecutionEngine::run() since there is an assert in ExecutionEngine::run  before laodValueFromTensor. However, for the DKKC8 opt, the filter is to be private. So I decided to add the ref result here and compare with it directly.

Reply:
@beicy I am not sure I understand why we need this. The compiler can take a network such as Resnet and VGG and perform the optimization that changes the layout internally. Why can't you just create a small convolutional network and let the compiler optimize it?  Any JIT test that uses convolutional neural network should run the DKKC path.

Reply:
Can we use `ExecutionEngine::run({}, {})` with both Interpreter and CPU backends? So `visibilityKind` is private for filter, but public for final results?

Reply:
@nadavrot I think the idea here was to test DKKC8 path (when `optimizeCPUConv()` triggers). We need >=64 channels to trigger it. Maybe it's a bit fragile, because it depends on `64` being hardcoded over there. Would it be better to test `libjit_conv.cpp`'s functions directly, instead of going through Graph and EE ?

Reply:
@artemrakhov We are going to have hundreds of different optimizations in different backends. I don't think that it makes sense to have a CPU-only test for this specific operator. Let's add a new JIT test like the other ones that we already have that checks the new grouped convolution. This is how we test the regular convolution.

Reply:
@artemrakhov Yes, we can use ExecutionEngine::run({}, {}) for both backends,  as long as the var is not loaded from tensor.